test-data/unit/check-flags.test

```diff
@@ -44,6 +44,46 @@ def f():
     1 + "str"
 [out]
 main:2: error: Function is missing a type annotation
+
+[case testDisallowUntypedDefsUntypedDecorator]
+# flags: --disallow-untyped-decorators
+def d(p):
+    return p
+
+@d  # E: Untyped decorator makes function "f" untyped
+def f(i: int) -> int:
+    return i
+
+[case testDisallowUntypedDefsUnresolvedDecorator]
+# flags: --disallow-untyped-decorators --ignore-missing-imports
+from nonexistent import d
+
+@d  # E: Untyped decorator makes function "f" untyped
+def f(i: int) -> int:
+    return i
+
+[case testDisallowUntypedDefsMultipleUntypedDecorators]
+# flags: --disallow-untyped-decorators --disallow-untyped-defs
```

Comment:
Hmm, my original thinking was to make sure to interacts well with `--disallow-untyped-defs` but I think that the flags don't have that much in common. I removed the test and added another one for multiple decorators.